blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/outputs/handlers/HttpMultiRollupsQueryHandlerIntegrationTest.java

```diff
@@ -51,54 +64,64 @@ public void testMultiplotQuery() throws Exception {
         JsonParser jsonParser = new JsonParser();
         JsonObject responseObject = jsonParser.parse(responseContent).getAsJsonObject();
 
-        String expectedResponse = "{\n" +
-                "  \"metrics\": [\n" +
-                "    {\n" +
-                "      \"unit\": \"unknown\",\n" +
-                "      \"metric\": \"3333333.G1s\",\n" +
-                "      \"data\": [\n" +
-                "        {\n" +
-                "          \"numPoints\": 1,\n" +
-                "          \"timestamp\": 1382400000,\n" +
-                "          \"latest\": 397\n" +
-                "        }\n" +
-                "      ],\n" +
-                "      \"type\": \"number\"\n" +
-                "    },\n" +
-                "    {\n" +
-                "      \"unit\": \"unknown\",\n" +
-                "      \"metric\": \"3333333.G10s\",\n" +
-                "      \"data\": [\n" +
-                "        {\n" +
-                "          \"numPoints\": 1,\n" +
-                "          \"timestamp\": 1382400000,\n" +
-                "          \"latest\": 56\n" +
-                "        }\n" +
-                "      ],\n" +
-                "      \"type\": \"number\"\n" +
-                "    }\n" +
-                "  ]\n" +
-                "}";
-        JsonObject expectedObject = jsonParser.parse(expectedResponse).getAsJsonObject();
-
-        Assert.assertEquals(expectedObject, responseObject);
-        EntityUtils.consume(query_response.getEntity());
```

Comment:
The EntityUtils.consume() got removed after your refactor. Is that useful for something? Or its just not needed anymore? It seems to be closing some InputStream.